docs/src/main/asciidoc/security-firebase.adoc

```diff
@@ -0,0 +1,65 @@
+:spring-security-ref: https://docs.spring.io/spring-security/reference/
+:spring-security-javadoc: https://docs.spring.io/spring-security/site/docs/current/api/org/springframework/security/
+
+[#security-firebase]
+== Firebase Authentication
+
+https://firebase.google.com/products/auth[Firebase Authentication] provides backend services, easy-to-use SDKs, and ready-made UI libraries to authenticate users to your link:https://firebase.google.com/[Firebase] app. It integrates tightly with other Firebase services and link:https://firebase.google.com/firebase-and-gcp[GCP services] including Cloud Storage, Cloud Firestore and Cloud Functions.
```

Comment:
I'd remove the "it integrates tightly" sentence. We've already seen that Firebase storage is different from GCP storage.

Reply:
hmm, right. My intension was to mention some GCP services can be accessed via firebase project authentication. But this seems to add more unnecessary confusion.